.prettierrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "arrowParens": "always",
+  "printWidth": 100,
```

Comment:
For me, I find this too short. On my laptop alone I can comfortably get to 150. With external monitors it is much higher.
The reason I point this out is that it will unnecessarily change a line and make it harder to read.
I think 120-130 is reasonable. 
But this is just my opinion and I will not hold up the PR for this.